docs/chart_repository_sync_example.md

```diff
@@ -38,7 +38,8 @@ Building synchronization state...
 Starting synchronization
 Would copy file://fantastic-charts/alpine-0.1.0.tgz to gs://fantastic-charts/alpine-0.1.0.tgz
 Would copy file://fantastic-charts/index.yaml to gs://fantastic-charts/index.yaml
-Are you sure you would like to continue with these changes?? [y/N]} y
+Are you sure you would like to continue with these changes?
+ [y/N]} y
```

Comment:
Can you move this line just after the question mark in the liner above separated by a space:
`Are you sure you would like to continue with these changes? [y/N]} y`